app/styles/_onboarding.scss

```diff
@@ -0,0 +1,155 @@
+//COLORS
+$c-gray--dark: #333333;
+$c-denim-blue: #00518e;
+
+//FONTS 
+$font-heading: "Playfair Display";
+$font-body: "OpenSans-Regular", sans-serif; 
+$font-body-bold: "OpenSans-Bold", sans-serif;
+
+$font-cool-heading: 'Monoton', cursive;
+$font-pixel-heading: 'Press Start 2P', monospace;
+$font-pixel-body: 'OpenSans', monospace;
```

Comment:
not as familiar with `wqxr-web-client` code base -- is it possible these font variables exist elsewhere?

Reply:
also, you can put these in `vars.scss`